backend/src/db/migrations/20230906150722_make_users_email_index.ts

```diff
@@ -0,0 +1,15 @@
+import { type Knex } from 'knex';
+
+async function up(knex: Knex): Promise<void> {
+  return await knex.raw(
+    'CREATE UNIQUE INDEX IF NOT EXISTS users_lowercase_email ON users (lower(email)); alter table users drop constraint users_email_unique;',
+  );
+}
+
+async function down(knex: Knex): Promise<void> {
+  return await knex.raw(
+    'DROP index IF EXISTS users_lowercase_email; ALTER TABLE users ADD CONSTRAINT user_email_unique UNIQUE (email);',
+  );
```

Comment:
Just to make queries more readable
```suggestion
  await knex.raw(
    'DROP INDEX IF EXISTS users_lowercase_email;'
  );
  await knex.raw(
    'ALTER TABLE users ADD CONSTRAINT user_email_unique UNIQUE (email);',
  );
```